src/SDKs/ServiceFabric/Management.ServiceFabric/Microsoft.Azure.Management.ServiceFabric.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.ServiceFabric</PackageId>
 		<Description>Microsoft Azure Management ServiceFabric Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.ServiceFabric</AssemblyName>
-		<Version>1.0.1-preview</Version>
```

Comment:
Looks like breaking changes are being introduced here, please set version to `1.1.0`
Also, if this is a stable release, please update the `PackageReleaseNotes`